API_Guidelines.md

```diff
@@ -73,15 +73,25 @@ Payload
 **Need documentation**. In the mean time [Finding Records](https://guides.emberjs.com/v2.10.0/models/finding-records/)
 
 ## POST
-`save()` will make a POST request *only* if the record is *new*.  
 
 ### Creating Records
 #### Request
 URL
 :   `apiHost.com/movies`  
 
 Ember Data Method
-:   `movie.save()`
+:  
+
+*`save()` will make a POST request only if the record is new.*
+
+```javascript
+let movie3 = get(this, 'store').createRecord('movie', { //create movie3 record
+    id: 3,
```

Comment:
are we setting ids on create requests? i don't think think id should be necessary since the backend will assign this.